pkg/api/dryrunclient.go

```diff
@@ -95,11 +101,15 @@ func (d *DryRunClient) ContainerUnpause(ctx context.Context, container string) e
 }
 
 func (d *DryRunClient) CopyFromContainer(ctx context.Context, container, srcPath string) (io.ReadCloser, moby.ContainerPathStat, error) {
-	return nil, moby.ContainerPathStat{}, ErrNotImplemented
+	rc := io.NopCloser(strings.NewReader(""))
+	if _, err := d.ContainerStatPath(ctx, container, srcPath); err != nil {
+		return rc, moby.ContainerPathStat{}, fmt.Errorf(" %s Could not find the file %s in container %s", DRYRUN_PREFIX, srcPath, container)
```

Comment:
I was going to suggest `s/file/path` but then realized this text comes from `moby/moby` 🙈